schemas/v3.0/schema.yaml

```diff
@@ -43,6 +43,14 @@ definitions:
       '^\$ref$':
         type: string
         format: uri-reference
+  SchemaReference:
+    type: object
+    required:
+      - $ref
+    patternProperties:
+      '^\$ref$':
+        type: string
+        format: uri-reference
```

Comment:
this could also just be

```yaml
SchemaReference:
  $ref: '#/definitions/Reference'
```

to avoid duplication. just having a separate schema to differentiate a SchemaReference vs a plain Reference is all that matters to me.